### PR TITLE
Add descriptions to my blog posts for improved SEO

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

File: _source/_posts/2017-08-22-build-an-ionic-app-with-user-authentication.md [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 tweets: 
     - "Learn how to build an @ionicframework app with user authentication →"
     - "Leverage OIDC and @okta to add authentication to your Ionic app →"
 ---
 
-With Okta and OpenID Connect (OIDC) you can easily integrate authentication into an Ionic application, and never have to build it yourself again. OIDC allows you to authenticate directly against the [Okta API](/), and this article shows you how to do just that in an Ionic application. I'll demo how to log in with OIDC redirect, using Okta's Auth SDK, and using OAuth with Cordova's in-app browser; user registration is omitted as the feature is still under active development. 
+With Okta and OpenID Connect (OIDC) you can easily integrate authentication into an Ionic application, and never have to build it yourself again. OIDC allows you to authenticate directly against the [Okta API](/product), and this article shows you how to do just that in an Ionic application. I'll demo how to log in with OIDC redirect, using Okta's Auth SDK, and using OAuth with Cordova's in-app browser; user registration is omitted as the feature is still under active development.
```
Comment (nbarbettini) [inline _source/_posts/2017-08-22-build-an-ionic-app-with-user-authentication.md]: This is throwing off travis, I think it needs a trailing slash: `/product/`

File: _source/_posts/2017-06-21-what-the-heck-is-oauth.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 layout: blog_post
 title: 'What the Heck is OAuth?'
 author: mraible
+description: "OAuth 2.0 is a standard that apps can use to provide client applications with secure delegated access. OAuth works over HTTPS and authorizes devices, APIs, servers, and applications with access tokens rather than credentials."
```
Comment (nbarbettini) [inline _source/_posts/2017-06-21-what-the-heck-is-oauth.md]: Maybe throw in a blurb for OpenID Connect here too? (to capture OIDC keywords)

File: _source/_posts/2017-08-08-secure-spring-microservices.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 layout: blog_post
 title: 'Secure a Spring Microservices Architecture with Spring Security, JWTs, Juiser, and Okta'
 author: mraible
+description: "This tutorial shows you how you can use Spring Security, Okta, and a few Java libraries to secure your microservices architecture. Not only that, but I'll show you how to secure *everything*, so even your backend services communicate securely."
```
Comment (nbarbettini) [inline _source/_posts/2017-08-08-secure-spring-microservices.md]: IDK how the Googles handles *markdown* - maybe just keep it as plain text?

File: _source/_posts/2017-08-08-secure-spring-microservices.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 layout: blog_post
 title: 'Secure a Spring Microservices Architecture with Spring Security, JWTs, Juiser, and Okta'
 author: mraible
-description: "This tutorial shows you how you can use Spring Security, Okta, and a few Java libraries to secure your microservices architecture. Not only that, but I'll show you how to secure *everything*, so even your backend services communicate securely."
+description: "This tutorial shows you how you can use Spring Security, Okta, and a few Java libraries to secure your microservices architecture. Not only that, but I'll show you how to secure everything, so even your backend services communicate securely."
```
Comment (rdegges) [inline _source/_posts/2017-08-08-secure-spring-microservices.md]: Maybe rewrite this slightly to: "... I'll also show you how to secure everything so even your backend services can communicate securely!"

File: _source/_posts/2017-08-22-build-an-ionic-app-with-user-authentication.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ tweets:
     - "Leverage OIDC and @okta to add authentication to your Ionic app →"
 ---
 
-With Okta and OpenID Connect (OIDC) you can easily integrate authentication into an Ionic application, and never have to build it yourself again. OIDC allows you to authenticate directly against the [Okta API](/product), and this article shows you how to do just that in an Ionic application. I'll demo how to log in with OIDC redirect, using Okta's Auth SDK, and using OAuth with Cordova's in-app browser; user registration is omitted as the feature is still under active development. 
+With Okta and OpenID Connect (OIDC) you can easily integrate authentication into an Ionic application, and never have to build it yourself again. OIDC allows you to authenticate directly against the [Okta API](/product/), and this article shows you how to do just that in an Ionic application. I'll demo how to log in with OIDC redirect, using Okta's Auth SDK, and using OAuth with Cordova's in-app browser; user registration is omitted as the feature is still under active development.
```
Comment (rdegges) [inline _source/_posts/2017-08-22-build-an-ionic-app-with-user-authentication.md]: With Okta and OpenID Connect (OIDC) you can easily integrate authentication into an Ionic application and never have to build it yourself again. OIDC allows you to authenticate directly against the [Okta API](/product/), and this article shows you how to do just that in an Ionic application. I'll demo how to log in with an OIDC redirect using Okta's Auth SDK as well as how to use OAuth with Cordova's in-app browser; user registration is omitted as the feature is still under active development.

**Pull request merged**